pkg/rancher/migrations.go

```diff
@@ -132,6 +141,33 @@ func createOrUpdateConfigMap(configMapClient controllerv1.ConfigMapClient, cm *v
 	return err
 }
 
+func versionTombstone(configMapController controllerv1.ConfigMapController) error {
+	cm, err := getConfigMap(configMapController, rancherVersionTombstoneConfig)
+	if err != nil || cm == nil {
+		return err
+	}
+
+	if !semver.IsValid(rancherversion.Version) {
```

Comment:
Do our hotfix releases follow valid semver? If not, do we need special logic here to handle those versions?

Reply:
Looking at the previous doc - they do follow semver, but they appear to be "pre-release semver". This means, according to the [semver spec](https://semver.org/#spec-item-11), they will sort as earlier than any released version. In turn I think this means that users won't be able to upgrade to hotfix version with this change - I think that's a large issue that we need to fix.